modules/installation-bare-metal-agent-installer-config-yaml.adoc

```diff
@@ -76,6 +76,31 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 <9> The cluster network provider Container Network Interface (CNI) plug-in to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN` .
 <10> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
 <11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
++
+[NOTE]
+====
+When you set the platform to `vSphere` or `baremetal`, you can configure IPv4 and IPv6 address endpoints for cluster nodes.
+
+.Example
+[source,yaml]
+----
+networking:
+  clusterNetwork:
+    - cidr: 172.21.0.0/16
+      hostPrefix: 23
+  machineNetwork:
+    - cidr: 192.168.11.0/24
+  networkType: OVNKubernetes
+  serviceNetwork:
+    - 172.22.0.0/16
+platform:
+  baremetal:
+    apiVIPs:
+    - 192.168.11.3
+    ingressVIPs:
+    - 192.168.11.4
+----
+====
```

Comment:
It seems like there should be a example for IPv6 or dual-stack (IPv4/IPv6) instead of just IPv4 here.

Reply:
my mistake, I was using the example from the TE slides.

Reply:
I have updated one example, if it looks good, then I will update the others . For dual-stack , there have two cidrs for the _machineNetwork_, _clusterNetwork_, and _serviceNetwork_.  

@bfournie ptal.

Reply:
@bfournie  the build is now passing. ptal I have implemented your feedback. Thanks